README.md

```diff
@@ -0,0 +1,91 @@
+
+# EMC_post
+
+The EMC_post package provides tools for post processing for NCEP
+models.
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
+project. It also serves as a standalone package that is distributed to
+the community. It is built outside of nceplibs for some UFS
+applications (e.g. SRW v1.0.0).
+
+## Authors
+
+NCEP/EMC Developers
+
+Code Manager: WenMeng
```

Comment:
Please add the changes as
Code Managers: Wen Meng, Huiya Chuang, Kate Fossell

Reply:
Done.